mw/i2c_service/controller/LSM6DSOX/controller.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file controller.cpp
+ * @author Mateusz Krajewski (matikrajek42@gmail.com)
+ * @brief 
+ * @version 0.1
+ * @date 2025-04-07
+ * 
+ * @copyright Copyright (c) 2025
+ * 
+ */
+#include <utility>
+#include "mw/i2c_service/controller/LSM6DSOX/controller.hpp"
+namespace srp {
+namespace i2c {
+namespace {
+    constexpr auto kCTRL2_G = 0x11;
```

Comment:
```suggestion
    static constexpr uint8_t kCTRL2_G = 0x11;
```
i tak dla pozostałych